classes/db/DAO.php

```diff
@@ -161,7 +161,7 @@ public function update($sql, $params = [], $callHooks = true, $dieOnError = true
             // this method is only called by a subclass. Results
             // in hook calls named e.g. "sessiondao::_updateobject"
             // (all lowercase)
-            $value = null;
```

Comment:
This is a long-standing behaviour dating back to our use of ADODB before we shifted the DB layer to Laravel, and it may or may not affect third-party plugins. This function is deprecated, so I'd rather leave it unaffected until it can be removed entirely.

Reply:
No complaints here, I just wanted to match the expected return :)